ui/src/app/components/genomics-extraction-table.tsx

```diff
@@ -220,19 +220,11 @@ export const GenomicsExtractionTable = fp.flow(withCurrentWorkspace())(
       >
         <div className='slim-scroll-bar'>
           <SwitchTransition>
-            <CSSTransition
-              key={!jobs}
+            <CSSTransition<undefined>
```

Comment:
Does anyone know what this transition does?

Reply:
This applies a 250ms fade transition between the loading, error, and success states. See the bottom of `genome-extraction-datatable.css` for the relevant CSS.

It's slick looking, but was this worth the code maintenance cost? Maybe add a comment to prevent us from spending another hour investigating this 😄

https://github.com/all-of-us/workbench/assets/31020403/0293c29b-4435-4934-a183-f85ca7fce252